packages/react-hooks/src/useBlockInterval.ts

```diff
@@ -2,14 +2,16 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import type { ApiPromise } from '@polkadot/api';
+import type { BabeGenesisConfiguration } from '@polkadot/types/interfaces/babe';
 
 import { useMemo } from 'react';
 
+import { A_DAY } from '@polkadot/react-hooks/useBlocksPerDays';
```

Comment:
The import here should be `import { A_DAY } from './useBlocksPerDays.js';`

Reply:
Fixed in [af37d65](https://github.com/polkadot-js/apps/pull/10457/commits/af37d6593521611490e1193a54ac7b48c57562bd).